### PR TITLE
Update to OpenPGP.js 5.3.1

### PR DESCRIPTION
**Pull request by tacerus**
Hi,

the current version of OpenPGP.js the program relies on is already relatively old. This patch updates the program to utilize the currently latest version of the library.

Let me know if you have any suggestions for improvement.

Best,
Georg

### CODE REVIEW

**Comment by toberndo** [comment]
Thanks for working on this! Yes, definitely time to bring OpenPGP.js up-to-date.

File: src/service/public-key.js [hunk 1]
```diff
@@ -78,8 +79,8 @@ class PublicKey {
     // if emails array is empty, all userIds of the key will be submitted
     if (emails.length) {
       // keep submitted user IDs only
-      key.userIds = key.userIds.filter(({email}) => emails.includes(email));
-      if (key.userIds.length !== emails.length) {
+      key.userIds = key.getUserIDs.filter(({email}) => emails.includes(email));
```
Comment (toberndo) [inline src/service/public-key.js:82]: Looks like this should be: `key.getUserIDs().filter`

Comment (tacerus) [reply src/service/public-key.js]: Thanks for the input - if you mean `key.getUserIDs().filter(({email}) => ......` that seems to work the same way, but you're right, it makes more sense with the parentheses.

**Comment by tacerus** [comment]
I unfortunately only noticed after submitting this PR that quite a lot of unit tests are not compatible with it. I'll try to look into those, however unfortunately I'm not that proficient with Mocha.

**Comment by toberndo** [comment]
Update to OpenPGP.js v5 has now landed with the new v4 release of the key server.

**Comment by tacerus** [comment]
Thank you, that's great news. Sorry for not having been able to follow up with this sooner.

**Pull request closed without merging**